src/samples/ChildBenefitsClaim/index.tsx

```diff
@@ -464,11 +464,11 @@ export default function ChildBenefitsClaim() {
 
       // TODO : Consider refactoring 'en_GB' reference as this may need to be set elsewhere
```

Comment:
Is this ToDo action still needed if so has it been recorded on Agile Studio?

Reply:
Will remove the Todo action.

While i'm sure there may be some value in refactoring this, it's currently not clear that this is going to be a constant across all projects, so will look into this if the logic is moved into a shared component.